README.md

```diff
@@ -4,11 +4,17 @@ A small pair of apps to enable paratransit riders to see the location and ETA of
 
 ## Development
 
-To start your Phoenix server:
+Install dependencies:
+
 
   * Run `mise install` to install the correct versions of Elixir/Erlang/NodeJS
   * Run `npm i --prefix assets` to install NodeJS packages
   * Run `mix setup` to install and setup dependencies
+  * Run `brew install libomp` to install a dynamic library that EXGBoost needs
```

Comment:
suggestion: this needs to be before `mix setup` so that it's included in the compiled dependency.